common/persistence/sql/cluster_metadata.go

```diff
@@ -44,6 +45,7 @@ type sqlClusterMetadataManager struct {
 var _ p.ClusterMetadataStore = (*sqlClusterMetadataManager)(nil)
 
 func (s *sqlClusterMetadataManager) ListClusterMetadata(
+	_ context.Context,
 	request *p.InternalListClusterMetadataRequest,
 ) (*p.InternalListClusterMetadataResponse, error) {
 	ctx, cancel := newExecutionContext()
```

Comment:
This PR does not include any behavior change. 
`newExecutionContext` will be removed in a follow up PR for actually using the passed in context.